src/components/appointments/Appointments.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  fetchAppointments,
+  selectAppointments,
+  selectAppointmentsStatus,
+  selectAppointmentsError,
+  deleteAppointment,
+} from '../../redux/appointmentsSlice';
+import Navigation from '../Navigation';
+
+function Appointments() {
```

Comment:
- [ ] You are including the patient's name in the "appointments" page, but every one of them is "John Doe":

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/905a049f-5a8c-403e-b98e-b352e317e929)

Kindly, make it so the user that made the appointment is shown their user name on that field. 👍